authorizer.py

```diff
@@ -142,10 +147,8 @@ def Redirect(self):
     logging.info("Redirecting %s to /welcome", self.email)
     self.handler.redirect("/welcome")
 
-  # TODO get rid of the unnecessary handler parameter
```

Comment:
Not just for GetLogoutUrl, but the entire authorizer?
Do you mean initialize authorizer on every page
this way
`auth = authorizer.Authorizer(self.request)`
instead of
`auth = authorizer.Authorizer(self)`

What is the harm in passing the handler? Is it simply not necessary or is there real harm (security risk or something)?